### PR TITLE
Add PHP-CS-Fixer level2 for Travis

### PR DESCRIPTION
**Pull request by lucasmichot**

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
:-1: from me. I hate the idea of running cs fixers on travis.

File: .travis.yml [hunk 1]
```diff
@@ -10,6 +10,11 @@ php:
 
 sudo: false
 
-install: travis_retry composer install --no-interaction --prefer-source
+install:
+  - wget http://get.sensiolabs.org/php-cs-fixer.phar
```
Comment (GrahamCampbell) [inline .travis.yml]: why are you doing this and requiring it through composer?

**Comment by lucasmichot** [comment]
Why @GrahamCampbell ?
Some other packages also do it : https://github.com/thephpleague/omnipay-common

**Comment by GrahamCampbell** [comment]
Travis should not fail because someone screwed up the cs. We need something separate for that.

**Comment by vlucas** [comment]
I actually don't think this is a terrible idea. It would prevent people from submitting code that does not conform to PSR-2, which is the goal of this. What else would you suggest for this, @GrahamCampbell ?

**Comment by GrahamCampbell** [comment]
@vlucas I still run cs checks on pull requests automatically, just on on travis. Symfony use fabbot.io, I use styleci. https://github.com/StyleCI.

**Pull request closed without merging**